Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -485,57 +485,57 @@ interface Tile {
 
 @Component({
   components: {
-    CommunicationComponent
-  }
+    CommunicationComponent,
+  },
 })
 export default class LandingComponent extends Vue {
   private icons: Icon[] = [
     {
       definition: faPills,
       label: "Medications",
-      active: true
-    },
-    {
-      definition: faSyringe,
-      label: "Vaccinations",
-      active: true
+      active: true,

```

Comment:
should the trailing commas be included?

Reply:
My lintfix likes to add them while manuel's removes them... what does yours do? Can update on my end.

Reply:
This seems to be a problem with lintfix. Mine removes them while sam's is adding them...